doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
```

Comment:
> Yes, I just wanted to be sure the example was valid when running make
> test-doc.

Make sense. I wonder if we can have multiple modes. It probably goes in
the direction of mocking the downloaders. @AlexandreAbraham might have an
idea on this.

> I can revert this change.

It would be good.